test/videojs-http-streaming.test.js

```diff
@@ -3109,7 +3109,7 @@ QUnit.skip('populates quality levels list when available', function(assert) {
   assert.ok(this.player.tech_.hls.qualityLevels_, 'added quality levels from video with source');
 });
 
-QUnit.test('configures eme if present on selectedinitialmedia', function(assert) {
+QUnit.test('configures eme for DASH if present on selectedinitialmedia', function(assert) {
```

Comment:
the changes in this test is due to how we're setting up EME now? Is it possible to make it so that the test doesn't change?

Reply:
I think the mocking in this test does need to be updated since the old one mocked objects that are no longer used to setup eme